src/Workspaces/Core/Portable/Formatting/Context/FormattingContext.cs

```diff
@@ -570,8 +570,8 @@ private AnchorData FindAnchorSpanOnSameLineAfterToken(TokenData tokenData)
 
         public bool IsWrappingSuppressed(TextSpan textSpan, bool containsElasticTrivia)
         {
-            // use edge exclusive version of GetSmallestContainingInterval
-            var data = _suppressWrappingTree.GetSmallestEdgeExclusivelyContainingInterval(textSpan.Start, textSpan.Length);
+            // use edge inclusive version of GetSmallestContainingInterval
+            var data = _suppressWrappingTree.GetSmallestEdgeInclusivelyContainingInterval(textSpan.Start, textSpan.Length);
```

Comment:
why?

Reply:
This is fallout of a root issue where wrapping suppressions are used for controlling spacing changes.

https://github.com/dotnet/roslyn/blob/af9741b029487cec82fc3232c6821a45e9f16d9c/src/Workspaces/Core/Portable/Formatting/Engine/AbstractFormatEngine.cs#L444

https://github.com/dotnet/roslyn/blob/af9741b029487cec82fc3232c6821a45e9f16d9c/src/Workspaces/Core/Portable/Formatting/Engine/AbstractFormatEngine.OperationApplier.cs#L196-L202

I modified the wrapping suppression to include the entire `spanBetweenTokens`, but it still didn't work. It didn't seem to make sense to have the exact span suppressed but not end up with a suppression, so I made this change.

Reply:
Thanks for the explanation!